geniza/corpus/migrations/0019_documenttype_display_label.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.8 on 2021-10-25 13:40
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("corpus", "0018_document_doctype_help_link"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="documenttype",
+            name="display_label",
+            field=models.CharField(blank=True, max_length=255),
```

Comment:
It irks me that django creates migrations for meta changes like help text, and I try to avoid creating unnecessary migrations, especially within a feature branch.

Please integrate this help text change to the initial migration, then delete 0021 and confirm it's no longer needed.

```suggestion
            field=models.CharField(
                blank=True,
                help_text="Optional label for display on the public site",
                max_length=255,
            ),
```